lib/data/src/text.rs

```diff
@@ -256,15 +258,15 @@ impl TextChange {
 
 
 
-// ===========================
-// === Change Notification ===
-// ===========================
+// ==========================
+// === Change Information ===
+// ==========================
 
-/// A notification about text change.
+/// An information about text change.
 ///
-/// In essence, it's `TextChange` with some additional useful information.
+/// In essence, it's `TextChange` with some additional useful data.
 #[derive(Clone,Debug,Shrinkwrap)]
-pub struct TextChangedNotification {
+pub struct TextChangedInfo {
```

Comment:
I understand your point, but I've done this change, because TextChangedNotification was very similar to notification::TextChanged, but those have a little different usages (one is from TextEditor to controllers, the other is in opposite direction.

Reply:
Finally we agreed on call, that we should one parametrized TextChanged structure.